doc/reference/animations.md

```diff
@@ -72,7 +72,7 @@ For example, a simple fade in/out effect can be done with this:
 
 ```xml
 <div>
-    <div t-if="state.flag" class="square" t-transition="fade">Hello</div>
+    <div class="square" t-transition="fade">Hello</div>
```

Comment:
It is better with the t-if. It emphases that there is a need of a switch between one state to another.